internal/services/costmanagement/export_resource_group_resource_test.go

```diff
@@ -127,6 +127,7 @@ resource "azurerm_resource_group_cost_management_export" "test" {
   recurrence_type              = "Monthly"
   recurrence_period_start_date = "%sT00:00:00Z"
   recurrence_period_end_date   = "%sT00:00:00Z"
+  file_format                  = "Csv"
```

Comment:
Since this property is Optional we shouldn't be setting this in the basic test config.
```suggestion
```